ibmcloud/README.md

```diff
@@ -112,72 +154,81 @@ You need to build a pod VM image for peer pod VMs. A pod VM image contains the f
+>     ```bash
+>     $ ibmcloud account users
+>     ```
+>     If command `ibmcloud account users` displays multiple user IDs, choose the user ID whose state is `ACTIVE`.
+> - `cos_service_instance_name` is the COS Service Instance Name, optional, default to value based on your `cluster_name` tfvar if not set, `${cluster_name}-cos-service-instance`.
+> - `cos_bucket_name` is the COS Bucket Name, optional, default to value based on your `cluster_name` tfvar if not set, `${cluster_name}-cos-bucket`.
 
-make verify
-```
 
-Note that creating a server from a new image may take long time. It typically takes about 10 minutes. From the second time, creating a server from the image takes one minute.
+> **Notes:**
+> - For uploading the pod VM image using Terraform, the COS Bucket must be a regional bucket in the same region (default `jp-tok`) as the VPC and VSIs.
+> - The `Operator` and `Console Admin` roles must be [assigned](https://cloud.ibm.com/docs/vpc?topic=vpc-vsi_is_connecting_console&interface=ui) to the user. The Terraform template will create the `Console Admin` role for the user `ibmcloud_user_id` is set to in the template `terraform.tfvars`.
 
-You can check the name and ID of the new image at [https://cloud.ibm.com/vpc-ext/compute/images](https://cloud.ibm.com/vpc-ext/compute/images). Alternatively, you can use the `ibmcloud` command to list your images as follows.
+Execute the following commands on your `development machine` to build, upload and verify the pod VM image.
 
+```bash
+$ cd ibmcloud/terraform/podvm-build
+$ terraform init
+$ terraform plan
+$ terraform apply
 ```
-ibmcloud is images --visibility=private
-```
+
+> **Notes:** 
+> - If your worker node is **s390x** based, the suffix of the created QCOW2 file for the custom image will be `-s390x` otherwise it will be `-amd64`.
+> - It typically takes about 15~20 minutes for task `Build peer pod VM image`
+> - It typically takes about 7~10 minutes for task `Push peer pod VM image to Cloud Object Store and verify the image`
+> - After all tasks finished, when you creating a server from the image it will only takes 1~5 minutes.
+> - You can check the name and ID of the new image at [https://cloud.ibm.com/vpc-ext/compute/images](https://cloud.ibm.com/vpc-ext/compute/images). Alternatively, you can use the `ibmcloud` command to list your images as follows.
+>    ```bash
+>    $ ibmcloud is images --visibility=private
+>    ```
 
 
 ## Install custom Kata shim
 
-The Ansible playbook automatically installs the custom Kata shim binary and its configuration file. If you want to rebuild the Kata shim, please follow the steps below.
+The Ansible playbook automatically installs the custom Kata shim binary and its configuration file to worker node. If you want to rebuild the Kata shim, please follow the steps below.
 
-```
-cd /root/kata-containers/src/runtime
-make $PWD/containerd-shim-kata-v2
-install containerd-shim-kata-v2 /usr/local/bin/
+```bash
+# cd /root/kata-containers/src/runtime
```

Comment:
Hey Da Li, I see that you've modified the commands to have the root shell prompt (`#`) in front. Whilst this is technically correct as we log into the worker VM as root user the [kata documentation guidelines](https://github.com/kata-containers/kata-containers/blob/main/docs/Documentation-Requirements.md#code-blocks) states:
> All lines beginning # should be comment lines, NOT commands to run as the root user.
I think the rest is that they appear as comment lines. I've run into this same problem myself when getting doc PRs reviewed and ended up using the non-root shell prompt (`$`) instead, which I think is the least bad option.

Reply:
fixed, use `$` replace `#` for the shell scripts